wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WebUtilities.java

```diff
@@ -502,9 +504,14 @@ public static void updateBeanValue(final WComponent component, final boolean vis
 		}
 
 		if (component instanceof Container) {
+
+			final List<WComponent> children = ((Container) component).getChildren();
+
+			Collections.reverse(children);
```

Comment:
This is a good example of how the original way of using indexes to access the list is better for performance.